scripts/sync-configs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "first make sure metrictank-sample.ini is up to date. its values should match the defaults used by metrictank. and comments should match the descriptions provided by metrictank help menus"
+echo "now we will run vimdiff to manually synchronize updates from sample config to other configs:"
+echo "try to make every config as closely resembling the sample config as possible, while retaining the customisations that makes each config unique"
```

Comment:
no native english speaker, but wouldn't that feel a little more natural:

> try to make every config resemble the sample config as closely as possible

and is `customisations` British spelling? my spell check says it should be `customizations`